templates/zerver/help/import-from-slack.md

```diff
@@ -1,149 +1,293 @@
+the transition easy for the members of your organization.
 
-This tool has been used to import Slack workspaces with 10,000 members
-and millions of messages. If you're planning on doing an import much
-larger than that, or run into performance issues when importing,
-[contact us](/help/contact-support) for help.
+The import will include:
 
-**Note:** You can only import a Slack workspace as a new Zulip
-organization. In particular, you cannot use this tool to import from Slack
-into an existing Zulip organization.
+* Your **message history**, including attachments and emoji reactions. Note that
+Slack's [data export service](https://slack.com/services/export) allows you to
+export all public channel messages, **including older messages that may no
+longer be searchable** under your Slack plan.
+* **Users**, including names, emails, roles, avatars, time zones, and custom profile fields
+* **Channels**, including all user subscriptions
+* **Custom emoji**
+* Your organization's **name** and **logo**
 
-## Import from Slack
+## Import process overview
 
-First, export your data from Slack.
+To import your Slack organization into Zulip, you will need to take the
+following steps, which are described in more detail below:
 
-!!! warn ""
+{start_tabs}
+
+1. Export your Slack data.
+
+2. Contact Zulip support to import your data into a Zulip Cloud organization,
+   *or* import it into your self-hosted Zulip server.
+
+3. Clean up after the Slack export.
 
-    **Note:** Only Slack owners and admins can export data from Slack.
-    See Slack's
-    [guide to data exports](https://get.slack.help/hc/en-us/articles/201658943-Export-data-and-message-history)
-    for more information.
+4. Get your organization's members started with Zulip!
 
-#### Get a Slack API token.
+{end_tabs}
+
+## Import your organization from Slack into Zulip
+
+### Export your Slack data
 
-It will be a long string starting with `xoxb-`.  It is required to
-fetch data that Slack doesn't include in their data exports, like
-email addresses.
+Slack's [data export
+service](https://slack.com/services/export) allows you to
+export all public channel messages, **including older messages that may no
+longer be searchable** under your Slack plan.
 
+Unfortunately, it may or may not be possible to export messages in your organization's
+private channels and direct messages:
+
+- Workspace Owners for Slack organizations on the Free or Pro plan can request
+  an export by [contacting Slack support](https://slack.com/help/requests/new),
+  which will be approved [“under limited
+  circumstances”](https://slack.com/help/articles/204897248-Guide-to-Slack-import-and-export-tools).
+
+- Owners for Slack organizations on the Business+ or Enterprise Grid plans can [apply
+  for access](https://slack.com/help/articles/204897248-Guide-to-Slack-import-and-export-tools) to a self-serve data export tool.
 
 {start_tabs}
 
-1. [Create a new Slack app](https://api.slack.com/apps).
```

Comment:
Having Steps 1 and 2 be "create a new Slack app" and "Create scope" seems "best" to me, as it immediately weeds out users without those permissions, who will inevitably get further down the list to suddenly discover they cannot proceed, instead of finding out earlier. I know the words "Make sure that you are..." are there, which is conceptually congruent, but easy to skip as there is no action here that will gate them. 

This isn't an objection, just rather a point of clarification: Do we want to give actionable steps here (previously 3 actionable steps), or make people read and here (currently 6 steps, but still only 3 actionable)?

Reply:
Hm, this comment made me think that we should give instructions for how to check that you're an admin, but I'm failing to find a Slack help page that answers this question.

Reply:
The reason I reordered the export at the top is that I wanted to get people going with the easier step.

Reply:
Which steps would you describe as not being actionable?